_includes/home/wtw-schedule.html

```diff
@@ -12,27 +10,42 @@ <h5 class="margin-top-0">Before submitting an offer you'll need to:</h5>
     <ul class="usa-list">
       <li>Register with SAM.gov</li>
       <li>Certify with Small Business Administration (SBA) for any set-asides</li>
-      <li>Understand the soliciation</li>
+      <li>Review and understand the contract schedule soliciation</li>
       <li>Complete self-led PDF trainings: Pathways to success and Readiness Assessment</li>
     </ul>
     <p><a href="javascript:void(0)" class="usa-link">Learn more about Getting on Schedule</a></p>
   </div>
 </div>
 
-<div class="grid-row grid-gap margin-top-6">
+<h4>Sell directly with Contract Schedules</h4>
```

Comment:
[question] This `h4` is identical to the one on line 1.  I'm wondering if there should only be one heading for the section, or if one of them should have different text.

Reply:
Changed it to "Becoming a vendor".